storage/innobase/row/row0import.cc

```diff
@@ -3115,11 +3124,15 @@ static dberr_t handle_instant_metadata(dict_table_t *table,
     return err;
 
   std::unique_ptr<byte[]> page_compress_buf(new byte[get_buf_size()]);
+  std::unique_ptr<byte[], decltype(&aligned_free)> crypt_tmp_frame(
+      static_cast<byte *>(aligned_malloc(physical_size, UNIV_PAGE_SIZE_MAX)),
+                          &aligned_free);
```

Comment:
Thanks for the review.

I didn't think this needs to be aligned at all so I just meant to be consistent with the other allocs in that section. I figured that maybe anything largish just gets page alignment to prevent some fragmentation via allocation offsets.

I'll update this.

Reply:
Is there an advantage of aligned_malloc to `CPU_LEVEL1_DCACHE_LINESIZE` vs. just `malloc` in this case?